src/Dropdown/index.jsx

```diff
@@ -131,14 +131,25 @@ class Dropdown extends React.Component {
   }
 
   render() {
+    const hasIcon = this.props.icon.url.trim();
+
     return (
       <div
         className={classNames([
           styles.dropdown,
-          { [styles.show]: this.state.open },
+          { [styles.show]: this.state.open, 'has-icon': hasIcon },
         ])}
         ref={(container) => { this.container = container; }}
       >
+        {hasIcon &&
+          <div className="icon-container">
+            <img
```

Comment:
rather than forcing this to be an img, when someone may want an `<Icon>` here, I think making a props of `iconElement` that just gets rendered here would make more sense. That gives the consumer the flexibility to pass in an image or an icon.